samples/highcharts/chart/height-inherited/demo.css

```diff
@@ -0,0 +1,3 @@
+#container {
+    height: 400px;
```

Comment:
[600 x 400 is default](https://github.com/highcharts/highcharts/blob/master/ts/Core/Chart/Chart.ts#L1397), so something different than 400 (600?) would be good to make sure, inheritance is always demonstrated and tested.